core/lexicon/en/lexicon.inc.php

```diff
@@ -25,7 +25,8 @@
 $_lang['lexicon_import_err_upload'] = 'An error occurred while trying to upload the Lexicon Topic file. Check your web server permissions in trying to upload to the tmp directory, and validate that the file you are trying to upload is a valid file.';
 $_lang['lexicon_management'] = 'Lexicon Management';
 $_lang['lexicon_management_desc'] = 'Here you can override any Lexicon Entries across Namespaces and their respective Topics. Just double-click on any Entry\'s value to override it. To add new Lexicon Entries or Topics, just create their Topic files in their Namespace\'s respective directory.<br /><b>Each Entry will be available via the [[%name]] placeholder.</b>';
-$_lang['lexicon_rlfb_msg'] = 'Successfully regenerated [[+num]] strings.';
-$_lang['reload_from_base'] = 'Revert All Core Entries';
-$_lang['reload_success'] = 'Successfully reloaded [[+total]] strings.';
+$_lang['lexicon_revert'] = 'Revert Entries';
+$_lang['lexicon_revert_confirm'] = 'Are you sure you want to revert the "[[+namespace]]" namespace > "[[+topic]]" topic > "[[+language]]" lexicon to its default strings?';
+$_lang['lexicon_revert_success'] = 'Reverted [[+total]] total string(s): [[+names]]';
+$_lang['lexicon_revert_error'] = 'No lexicon entries found to revert.';
```

Comment:
It's not really an error, so consider this key name change:
```suggestion
$_lang['lexicon_revert_none_found'] = 'No lexicon entries found to revert.';
```

Reply:
I don't think it makes much sense. There is a pair `error \ success`, everything is clear.
Otherwise, we will have to change `lexicon_revert_error` in `lexicon_revert_none_found`, and `lexicon_revert_success` then in `lexicon_revert_all_found` =) ? Sophistry...